src/main/java/propertyFiles/citation.properties

```diff
@@ -345,7 +346,7 @@ controlledvocabulary.language.galician=Galician
 controlledvocabulary.language.georgian=Georgian
 controlledvocabulary.language.german=German
 controlledvocabulary.language.greek_(modern)=Greek (modern)
-controlledvocabulary.language.guarani=Guaran�
+controlledvocabulary.language.guarani=Guaraní
```

Comment:
These shouldn't be getting changed. In github, the old and new versions looks the same, so I can only guess that some encoding change has happened.

Reply:
@qqmyers yes, encoding change.

Reply:
Please remove the encoding changes. They are not related to cstr.

Reply:
I have modified the file code set, please pull the code again.

Reply:
@cheneyfeng3 thanks, looks much better. I just left a comment below.